src/config/syscheck-config.c

```diff
@@ -1458,6 +1585,15 @@ int Read_Syscheck(const OS_XML *xml, XML_NODE node, void *configp, __attribute__
                             os_free(tag);
                             return OS_INVALID;
                         }
+                    } else if (strcmp(node[i]->attributes[j], xml_report_changes) == 0) {
+                        if (strcmp(node[i]->values[j], "yes") == 0) {
+                            opts |= CHECK_SEECHANGES;
+                        } else if (strcmp(node[i]->values[j], "no") == 0) {
+                            opts &= ~ CHECK_SEECHANGES;
+                        } else {
+                            mwarn(FIM_INVALID_REG_OPTION_SKIP, node[i]->values[j], node[i]->attributes[j], node[i]->content);
```

Comment:
The memory pointed by `tag` is not freed.
This is the only bug reported by scan-build.
```suggestion
                            mwarn(FIM_INVALID_REG_OPTION_SKIP, node[i]->values[j], node[i]->attributes[j], node[i]->content);
                            os_free(tag);
```